phe/paillier.py

```diff
@@ -87,8 +88,9 @@ def __init__(self, n):
         self.max_int = n // 3 - 1
 
     def __repr__(self):
-        nsquare = self.nsquare.to_bytes(1024, 'big')
-        g = self.g.to_bytes(1024, 'big')
+        digest_bytes = max(1024, round(math.log2(self.nsquare)))
```

Comment:
are you mixing bits and bytes here? `floor(log2(n)) + 1` gives you the number of bits to represent `n`. `digest_bytes` suggests that you want to have the number of bytes to represent `n`.